README.md

```diff
@@ -107,31 +116,48 @@ Assuming the repo is in your home directory
 ```
 cd ~/usb-sd-creator
 
-./osx_build.sh
-```
-
-If building again cleanup using:
-```
-./osx_clean.sh
+cmake -S . -B build -D CMAKE_PREFIX_PATH="/Users/$USER/Qt/6.7.2" -D QT_DIR="~/Qt/6.7.2/macos/lib/cmake/Qt6" -D Qt6CoreTools_DIR="~/Qt/6.7.2/macos/lib/cmake/Qt6CoreTools" && cmake --build build
```

Comment:
@kambala-decapitator so I had to run this crazy long cmake command to get things to build, supplying two extra defines. Maybe just a quirk of AQT, or maybe I'm missing a package somewhere, not sure.

Reply:
you made a small mistake in CMAKE_PREFIX_PATH, correct command is:

    cmake -S . -B build -D CMAKE_PREFIX_PATH="/Users/$USER/Qt/6.7.2/macos"

and to build release, you can do simply `cmake --preset release && cmake --build --preset release`

Reply:
Magic, updated and add both to README, thanks!